content/spatial/wms.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "Using WMS service in R"
+date: 2018-02-19T12:09:22+01:00
+---
+
+```{r include = FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = TRUE
+)
+```
+
+WMS stands for [Web Map Service](https://en.wikipedia.org/wiki/Web_Map_Service). The service provides prerendered tiles at different scales. This makes it useful to include them as background images in maps. 
+
+There is currently only one example on this page. We plan to add more examples in the future. Feel free to add more examples through a pull request at https://github.com/inbo/tutorials/pulls. Or create an issue at https://github.com/inbo/tutorials/issues.
+
+```{r}
+wms_grb <- "https://geoservices.informatievlaanderen.be/raadpleegdiensten/GRB-basiskaart/wms"
```

Comment:
Could you refer to the reference page of Geopunt, http://www.geopunt.be/catalogus/webservicefolder/aa04ae22-2297-98c3-1ffd-3440-5aff-bd2c-8a0cc151 and make clear to the user that the download service itself is defined by the 
```
Service URL: https://geoservices.informatievlaanderen.be/raadpleegdiensten/GRB-basiskaart/wms?
```